Detectors/ITSMFT/ITS/postprocessing/studies/macros/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+# Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+# See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+# All rights not expressly granted are reserved.
+#
+# This software is distributed under the terms of the GNU General Public
+# License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+#
+# In applying this license CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization
+# or submit itself to any jurisdiction.
+
+o2_add_test_root_macro(
+  PostTrackExtension.C
+  PUBLIC_LINK_LIBRARIES ROOT::Hist ROOT::RIO ROOT::Core ROOT::Gpad
```

Comment:
Maybe you need:
```suggestion
  PUBLIC_LINK_LIBRARIES ROOT::Hist ROOT::RIO ROOT::Core ROOT::Gpad fmt::fmt
```

Reply:
Curiously, it still complains about it.
```
cling::DynamicLibraryManager::loadLibrary(): dlopen(libfmt.10.1.0.dylib, 0x0009): tried: 'libfmt.10.1.0.dylib' (no such file), '/System/Volumes/Preboot/Cryptexes/OSlibfmt.10.1.0.dylib' (no such file), '/Volumes/build/alice-ci-workdir/o2/sw/osx_arm64/ROOT/v6-32-06-alice1-local3/lib/../lib/libfmt.10.1.0.dylib' (no such file), '/Volumes/build/alice-ci-workdir/o2/sw/osx_arm64/ROOT/v6-32-06-alice1-local3/bin/../lib/libfmt.10.1.0.dylib' (no such file), '/usr/lib/libfmt.10.1.0.dylib' (no such file, not in dyld cache), 'libfmt.10.1.0.dylib' (no such file)
cling::DynamicLibraryManager::loadLibrary(): dlopen(/Volumes/build/alice-ci-workdir/o2/sw/BUILD/e89aa9f3d18dea00fd76f985216d2b4e905e7c6c/O2/compiled_macros/Volumes/build/alice-ci-workdir/o2/sw/SOURCES/O2/13568/0/Detectors/ITSMFT/ITS/postprocessing/studies/macros/PostTrackExtension_C.so, 0x0009): Library not loaded: @rpath/libfmt.10.dylib
  Referenced from: <7E867859-82DA-3B84-9471-92E6AA6D43A1> /Volumes/build/alice-ci-workdir/o2/sw/BUILD/e89aa9f3d18dea00fd76f985216d2b4e905e7c6c/O2/compiled_macros/Volumes/build/alice-ci-workdir/o2/sw/SOURCES/O2/13568/0/Detectors/ITSMFT/ITS/postprocessing/studies/macros/PostTrackExtension_C.so
  Reason: tried: '/Volumes/build/alice-ci-workdir/o2/sw/osx_arm64/ROOT/v6-32-06-alice1-local3/lib/libfmt.10.dylib' (no such file), '/System/Volumes/Preboot/Cryptexes/OS/Volumes/build/alice-ci-workdir/o2/sw/osx_arm64/ROOT/v6-32-06-alice1-local3/lib/libfmt.10.dylib' (no such file), '/Volumes/build/alice-ci-workdir/o2/sw/osx_arm64/ROOT/v6-32-06-alice1-local3/lib/libfmt.10.dylib' (no such file), '/System/Volumes/Preboot/Cryptexes/OS/Volumes/build/alice-ci-workdir/o2/sw/osx_arm64/ROOT/v6-32-06-alice1-local3/lib/libfmt.10.dylib' (no such file), '/Volumes/build/alice-ci-workdir/o2/sw/osx_arm64/ROOT/v6-32-06-alice1-local3/lib/../lib/libfmt.10.dylib' (no such file), '/Volumes/build/alice-ci-workdir/o2/sw/osx_arm64/ROOT/v6-32-06-alice1-local3/bin/../lib/libfmt.10.dylib' (no such file)
ld: Undefined symbols:
```

Reply:
Well correct to link it nonetheless but yes it did not fix it and stayed the same

Reply:
@ktf Sorry to bother you but do you have an idea what I am missing in the linking?